modules/admin/proc-rhdh-deploy-aks-using-operator.adoc

```diff
@@ -85,6 +85,45 @@ kubectl -n <your_namespace> apply -f rhdh-ingress.yaml
 ----
 --
 
+. Create a ConfigMap named `app-config-rhdh` containing the {product-short} configuration using the following template:
```

Comment:
we can call this an example rather than specifying it as a template. because templates have different meaning in different contexts. I see a completely different section for templates here: https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-217/admin-rhdh/#assembly-admin-templates.

better to avoid any confusion.

```suggestion
. Create a ConfigMap named `app-config-rhdh` containing the {product-short} configuration as shown in the following example:
```

Reply:
Alternate suggestion: Create a ConfigMap named `app-config-rhdh` containing the {product-short} configuration by using the following YAML code:

Reply:
used example. Done